### ISSUE TITLE
Crash when linting test.check

### ISSUE THREAD

**Issue by pgj** [borkdude/clj-kondo#181]
`clj-kondo` gives a `NullPointerException` when it is run for the latest `master` of `test.check`:

```sh
$ git clone -q https://github.com/clojure/test.check
$ clj-kondo --lint test.check
Unexpected error. Please report an issue.
java.lang.NullPointerException
	at clojure.lang.Numbers.ops(Numbers.java:1068)
	at clojure.lang.Numbers.gt(Numbers.java:259)
	at clj_kondo.impl.linters$lint_calls$iter__3215__3225$fn__3226$iter__3217__3227$fn__3228$iter__3219__3229$fn__3230.invoke(linters.clj:238)
	at clojure.lang.LazySeq.sval(LazySeq.java:42)
	at clojure.lang.LazySeq.seq(LazySeq.java:51)
	at clojure.lang.RT.seq(RT.java:531)
	at clojure.core$seq__5387.invokeStatic(core.clj:137)
	at clj_kondo.impl.linters$lint_calls$iter__3215__3225$fn__3226$iter__3217__3227$fn__3228.invoke(linters.clj:200)
	at clojure.lang.LazySeq.sval(LazySeq.java:42)
	at clojure.lang.LazySeq.seq(LazySeq.java:51)
	at clojure.lang.RT.seq(RT.java:531)
	at clojure.core$seq__5387.invokeStatic(core.clj:137)
	at clj_kondo.impl.linters$lint_calls$iter__3215__3225$fn__3226.invoke(linters.clj:200)
	at clojure.lang.LazySeq.sval(LazySeq.java:42)
	at clojure.lang.LazySeq.seq(LazySeq.java:51)
	at clojure.lang.RT.seq(RT.java:531)
	at clojure.core$seq__5387.invokeStatic(core.clj:137)
	at clojure.core$dorun.invokeStatic(core.clj:3133)
	at clojure.core$doall.invokeStatic(core.clj:3148)
	at clojure.core$doall.invoke(core.clj:3148)
	at clj_kondo.main$main.invokeStatic(main.clj:317)
	at clj_kondo.main$main.doInvoke(main.clj:289)
	at clojure.lang.RestFn.applyTo(RestFn.java:137)
	at clojure.core$apply.invokeStatic(core.clj:665)
	at clojure.core$apply.invoke(core.clj:660)
	at clj_kondo.main$_main$fn__5821.invoke(main.clj:338)
	at clj_kondo.main$_main.invokeStatic(main.clj:338)
	at clj_kondo.main$_main.doInvoke(main.clj:336)
	at clojure.lang.RestFn.applyTo(RestFn.java:137)
	at clj_kondo.main.main(Unknown Source)
$ clj-kondo --version
clj-kondo v2019.05.12-alpha
```

**Comment by borkdude** [comment]
Thanks for the report. I can also reproduce this with the newest release of clj-kondo. I'll look into it.

**Comment by borkdude** [comment]
@pgj Are you using the Mac or Linux binary? I can provide you with a fixed binary for testing.

**Comment by pgj** [comment]
Mac.

**Comment by borkdude** [comment]
@pgj It should be fixed in this version:

https://2621-176829714-gh.circle-artifacts.com/0/release/clj-kondo-2019.05.20-alpha-SNAPSHOT-macos-amd64.zip

**Comment by pgj** [comment]
Thanks @borkdude, I can confirm that it works for me too.

**Comment by borkdude** [comment]
Fixed with 48e5bf8d563359c111c299d1efe4d39b961e5b3a. 

Please leave this issue open, I'll close it after the next release.
